metis-indexing/src/main/java/eu/europeana/indexing/SettingsConnectionProvider.java

```diff
@@ -122,10 +124,17 @@ private static CloudSolrClient setUpCloudSolrConnection(IndexingSettings setting
     builder.withLBHttpSolrClient(httpSolrClient);
 
     // Set up Zookeeper connection
-    LOGGER.info("Connecting to Zookeeper hosts: [{}] with chRoot [{}] and default connection [{}].",
-        hosts.stream().collect(Collectors.joining(", ")), chRoot, defaultCollection);
+    LOGGER.info(
```

Comment:
Let's add the check before doing the logging.
if (LOGGER.isInfoEnabled())
For the other two calls in this file too.